src/components/Progressbar.tsx

```diff
@@ -0,0 +1,103 @@
+import classNames from 'classnames';
+import { FC, PropsWithChildren } from 'react';
+
+type Color = 'gray' | 'blue' | 'red' | 'green' | 'yellow' | 'indigo' | 'purple';
+type Size = 'sm' | 'md' | 'lg' | 'xl';
+
+export type ProgressbarProps = PropsWithChildren<{
+  className?: string;
+  progress?: number;
```

Comment:
I think the progress prop should be required.

Reply:
Made the progress props required